lib/archives_sidebar/lib/archives_sidebar.rb

```diff
@@ -30,11 +30,11 @@ def parse_request(_contents, _params)
     article_counts = Content.find_by_sql(["select count(*) as count, #{date_func} from contents where type='Article' and published = ? and published_at < ? group by year,month order by year desc,month desc limit ? ", true, Time.now, count.to_i])
 
     @archives = article_counts.map do |entry|
-      month = (entry.month.to_i%12)+1
+      month = (entry.month.to_i%12)
```

Comment:
What does the deletion of the '+1' achieve go sold issue #2? Please describe your solution.